<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-G67TU0C1" failures="1" tests="1" name="com.herokuapp.theinternet.javascripterrortest.JavascriptErrorTests" time="4.620" errors="0" timestamp="2023-09-25T16:59:54 CST" skipped="0">
  <testcase classname="com.herokuapp.theinternet.javascripterrortest.JavascriptErrorTests" name="javascriptLoadingPageTest" time="4.620">
    <failure message="The following asserts failed:
	Severe error: https://298279967.log.optimizely.com/event?a=298279967&amp;d=298279967&amp;y=false&amp;n=https%3A%2F%2Fthe-internet.herokuapp.com%2F&amp;u=oeu1695682793190r0.887715345556072&amp;wxhr=true&amp;t=1695682793192&amp;f=298349752,318188263 - Failed to load resource: net::ERR_NAME_NOT_RESOLVED,
	Severe error: https://the-internet.herokuapp.com/favicon.ico - Failed to load resource: the server responded with a status of 404 (Not Found),
	Severe error: https://298279967.log.optimizely.com/event?a=298279967&amp;d=298279967&amp;y=false&amp;n=engagement&amp;g=298283957&amp;u=oeu1695682793190r0.887715345556072&amp;wxhr=true&amp;t=1695682794133&amp;f=298349752,318188263 - Failed to load resource: net::ERR_NAME_NOT_RESOLVED,
	Severe error: https://the-internet.herokuapp.com/javascript_error 6:51 Uncaught TypeError: Cannot read properties of undefined (reading &#039;xyz&#039;)" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Severe error: https://298279967.log.optimizely.com/event?a=298279967&d=298279967&y=false&n=https%3A%2F%2Fthe-internet.herokuapp.com%2F&u=oeu1695682793190r0.887715345556072&wxhr=true&t=1695682793192&f=298349752,318188263 - Failed to load resource: net::ERR_NAME_NOT_RESOLVED,
Severe error: https://the-internet.herokuapp.com/favicon.ico - Failed to load resource: the server responded with a status of 404 (Not Found),
Severe error: https://298279967.log.optimizely.com/event?a=298279967&d=298279967&y=false&n=engagement&g=298283957&u=oeu1695682793190r0.887715345556072&wxhr=true&t=1695682794133&f=298349752,318188263 - Failed to load resource: net::ERR_NAME_NOT_RESOLVED,
Severe error: https://the-internet.herokuapp.com/javascript_error 6:51 Uncaught TypeError: Cannot read properties of undefined (reading 'xyz')
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.herokuapp.theinternet.javascripterrortest.JavascriptErrorTests.javascriptLoadingPageTest(JavascriptErrorTests.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- javascriptLoadingPageTest -->
  <system-out/>
</testsuite> <!-- com.herokuapp.theinternet.javascripterrortest.JavascriptErrorTests -->
